code/addons/test/src/postinstall.ts

```diff
@@ -53,6 +55,34 @@ export default async function postInstall(options: PostinstallOptions) {
   // if Vitest is installed, we use the same version to keep consistency across Vitest packages
   const vitestVersionToInstall = vitestVersionSpecifier ?? 'latest';
 
+  if (info.frameworkPackageName === '@storybook/nextjs') {
+    const out = await prompts({
+      type: 'confirm',
+      name: 'migrateToExperimentalNextjsVite',
+      message: `To use this addon, you should migrate to use @storybook/experimental-nextjs-vite, do you want to migrate?`,
```

Comment:
You should check this part of the code below which already handles installing the package:
https://github.com/storybookjs/storybook/blob/13e54a028b9a4005142b113693e839218ca9c612/code/addons/test/src/postinstall.ts#L205

You should also check about the messaging, making it clear to the user why and provide a link to the docs (like the code I referred)

Reply:
Okay, I tried to improve the messaging. What do you think?